test/integration/targets/sysctl/tasks/main.yml

```diff
@@ -125,15 +147,18 @@
     value: 1
     sysctl_set: yes
     reload: no
+  when: sysctl_cmd_supported
```

Comment:
Why is this needed? The existing tests pass without it and should only be updating `sysctl.conf` without writing the new values with `sysctl`.